common/logger/README.md

```diff
@@ -60,8 +60,25 @@ This signature of the function is showed below. The `JsonValue` is a `enum` from
 pub fn metrics(name: &str, mut content: JsonValue)
 ```
 
+## Structured Event Log With TraceId Included
+
+Structured event log api provide a convenient way to log structured json data. It's signature is provided as below:
+
+```rust
+pub fn log(level: Level, module: &str, event: &str, ctx: &Context, mut msg: JsonValue)
+```
+
+`module` should be your component name, `event` is just event name, better begin with 4 chars with 4 digits
+to identify this event. `Context` is used to extract trace id. `msg` is `JsonValue` which is same as `metrics`.
+
+Useage example:
+
+```rust
+common_logger::log(Level::Info, "network", "netw0001", &ctx, json!{"music": "beautiful world"});
```

Comment:
The return type of `json!()` is `Value`, however, the `log()` method needs `JsonValue`.

Reply:
Oh, ```common-logger``` uses ```json``` not ```serde_json```.
Replace ```json!``` with ```common_logger::object!```.